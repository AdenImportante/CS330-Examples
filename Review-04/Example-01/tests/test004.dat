#! /bin/sh

# Repeat test003 with attention to formatting

# Consider all whitespace and letter-case (upper/lower)
./storage test003.in 4 