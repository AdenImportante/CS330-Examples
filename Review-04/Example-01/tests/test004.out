Processing Log:
 (S) Shovel
 (S) Pickaxe

Player Storage Summary:
 -Used  50% of 4 slots
  Nme: Shovel
  Dur: 20
  Spd: 3
  Mtl: Gold
  Mdr: Unbreaking (Lvl 2)

  Nme: Pickaxe
  Dur: 10
  Spd: 3
  Mtl: Stone
  Mdr: Efficiency (Lvl 2)

